README.md

```diff
@@ -1,2 +1,14 @@
-# lsstinstall
+lsstinstall
+==========
+
 Install eups distribution from eups.lsst.codes
+
+The tool (lsstinstall) is available in the bin folder,
+and can be used as is after cloning this repository.
+Please refer to bin/README.md.
+
+The tool is distributed via conda package. A draft recipe is in the recipe folder.
```

Comment:
The recipe doesn't belong here, I don't think.  It will add confusion once it's turned into a feedstock.

Reply:
I remove the mention to the recipe in the README.